src/lib/toaster.svelte.ts

```diff
@@ -0,0 +1,51 @@
+import { writable } from 'svelte/store';
+
+export type ToastObj = {
+	message: string;
+	type: 'success' | 'error' | 'info' | 'warning';
+	timeout: number; // Time before the toast is removed in ms
+	dismissable: boolean;
+	auto_dismiss: boolean; // If the toast should be removed after the timeout
+	initial_timeout: number; // The initial timeout
+};
+
+function timeout(toast: ToastObj) {
+	if (toast.timeout >= 0) {
+		setTimeout(() => timeout(toast), 1000);
+		toast.timeout -= 1000;
+	}
+	if (toast.auto_dismiss && toast.timeout < 0) removeToast(toast);
```

Comment:
```suggestion
	if (toast.auto_dismiss && toast.timeout < 0) {
	    removeToast(toast);
	}
```

Je préfère qu'on garde ce formatage partout (donc systématiquement les curly)